Content.Shared/Contraband/ContrabandSystem.cs

```diff
@@ -55,10 +57,23 @@ private void OnExamined(Entity<ContrabandComponent> ent, ref ExaminedEvent args)
         using (args.PushGroup(nameof(ContrabandComponent)))
         {
             var severity = _proto.Index(ent.Comp.Severity);
+            var jobList = new List<string>();
             if (severity.ShowDepartments && ent.Comp is { AllowedDepartments: not null })
             {
+                if (ent.Comp is { AllowedJobs: not null })
+                {
+                    jobList = ent.Comp.AllowedJobs.Select(p => Loc.GetString($"{p.Id}")).ToList();
+                }
+
                 // TODO shouldn't department prototypes have a localized name instead of just using the ID for this?
-                var list = ContentLocalizationManager.FormatList(ent.Comp.AllowedDepartments.Select(p => Loc.GetString($"department-{p.Id}")).ToList());
+                var departmentList = ent.Comp.AllowedDepartments.Select(p => Loc.GetString($"department-{p.Id}")).ToList();
+
+                //creating a combined list of jobs and departments for the restricted text
+                foreach (var job in jobList)
+                {
+                    departmentList.Add(job);
+                }
```

Comment:
```suggestion
                departmentList.AddRange(jobList);
```
did not test this but I think this function does the same